Package.swift

```diff
@@ -48,7 +48,7 @@ let package = Package(
     .package(
       name: "AppCheck",
       url: "https://github.com/google/app-check.git",
-      .branch("main")),
+      .revision("023abd45d302912a4f8387de7ab2b741659faca6")),
```

Comment:
It seems like you can specify a tag in place of a branch. This may be easier to track (if it works -- not tested):
```suggestion
      .branch("CocoaPods-0.1.0-alpha.6")),
```
Either way I'll try to keep you posted when there's a breaking change. It'll be easier when we tag the first release.

Reply:
Good point regarding the tag. I think in this case that sha is for the release in question.

Reply:
I'll change to the tag because it's more clear.